setup_dev_container.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR="$SOURCE_DIR/.travis"
+pushd $SCRIPT_DIR
+
+source $SCRIPT_DIR/travis-common.inc
+
+set -e
+set -x
+
+HOME=${HOME-/}
+
+release=$($SCRIPT_DIR/get_rawhide_version.py)
```

Comment:
```suggestion
release=${1:-$($SCRIPT_DIR/get_rawhide_version.py)}
```
It would be great to include a way to request a release other than rawhide.

Reply:
Oh, whoops. I meant to get around to that and forgot. Thanks for catching that.